nixos/modules/services/misc/sdrplay.nix

```diff
@@ -29,7 +29,9 @@ with lib;
         RestartSec = "1s";
       };
     };
+
     services.udev.packages = [ pkgs.sdrplay ];
 
+    boot.blacklistedKernelModules = [ "sdr_msi3101" "msi001" "msi2500" ];
```

Comment:
```suggestion
    # see https://www.sdrplay.com/community/viewtopic.php?t=2169#p6468
    boot.blacklistedKernelModules = [ "sdr_msi3101" "msi001" "msi2500" ];
```

Reply:
Better to also add notice and same link to `enable`